frame/system/src/lib.rs

```diff
@@ -232,6 +233,19 @@ pub mod pallet {
 			type BlockWeights = ();
 			type BlockLength = ();
 			type DbWeight = ();
+			#[verbatim]
+			type RuntimeEvent = ();
+			#[verbatim]
+			type RuntimeOrigin = ();
+			#[verbatim]
+			type RuntimeCall = ();
+			#[verbatim]
+			type Block = ();
+			#[verbatim]
+			type PalletInfo = ();
+			type BaseCallFilter = frame_support::traits::Everything;
+			type BlockHashCount = ConstU64<10>;
```

Comment:
Will these lines also be copied over?  
Then it would make sense to use the absolute path, or?